settings/dev.settings.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please delete the settings.json file, ignore it in .gitignore file

Reply:
Ok.